packages/modules/viewer-react/src/hooks/useUiProviders.tsx

```diff
@@ -2,21 +2,28 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-
+import { TreeWidgetUiItemsProvider } from "@bentley/tree-widget-react/lib/components/TreeWidgetUiItemsProvider";
 import { UiItemsManager, UiItemsProvider } from "@bentley/ui-abstract";
 import { useEffect } from "react";
 
 export function useUiProviders(uiProviders?: UiItemsProvider[]): void {
   useEffect(() => {
-    if (uiProviders) {
-      uiProviders.forEach((uiProvider) => {
-        UiItemsManager.register(uiProvider);
+    const defaultProviders: UiItemsProvider[] = [
+      new TreeWidgetUiItemsProvider(),
+    ];
+
+    const _uiProviders = uiProviders
```

Comment:
I would not use an underscore here. Instead rename the arg to something like "customUiProviders" and keep the const as simply "uiProviders"